packages/components/src/Tabs/Tabs.tsx

```diff
@@ -1,60 +1,57 @@
 import React from 'react';
 import { useTabList } from '@react-aria/tabs';
-import {
-  Box,
-  ThemeExtensionsWithParts,
-  useComponentStyles,
-} from '@marigold/system';
+import { cn, createVar, useClassNames } from '@marigold/system';
 import { useTabListState } from '@react-stately/tabs';
 import { useRef } from 'react';
 import { AriaTabListProps } from '@react-types/tabs';
 
 import { Tab } from './Tab';
 import { TabPanel } from './TabPanel';
-import { TabContext } from './Context';
 import { Item } from '@react-stately/collections';
+import { TabContext } from './Context';
 
-export interface TabsThemeExtension
-  extends ThemeExtensionsWithParts<'Tabs', ['tabs', 'tab', 'tabPanel']> {}
 interface TabsProps
   extends Omit<AriaTabListProps<object>, 'orientation' | 'isDisabled'> {
-  gap?: number;
+  gap?: string;
```

Comment:
you can use for this also `gapSpace` which is in `style-props.tsx`

Reply:
Yeah it is better, now it is added